llvm/lib/Transforms/Scalar/SimplifyCFGPass.cpp

```diff
@@ -61,6 +61,10 @@ static cl::opt<bool> UserSwitchRangeToICmp(
     cl::desc(
         "Convert switches into an integer range comparison (default = false)"));
 
+static cl::opt<bool> UserSwitchToSelect(
+    "switch-to-select", cl::Hidden, cl::init(false),
+    cl::desc("Convert switches into icmp + select (default = false)"));
+
```

Comment:
do we really need new cl::opts for this? Can just be a pass parameter?

Reply:
This is just for testing purposes, I guess in one point we will instantiate a pass with enable by default...